server/storage/database_test.go

```diff
@@ -440,3 +441,43 @@ func TestDBCheckHealthSucceeds(t *testing.T) {
 	err = dbStore.CheckHealth()
 	require.NoError(t, err)
 }
+
+func TestDBGetChecksum(t *testing.T) {
+	tempBaseDir, err := ioutil.TempDir("", "notary-test-")
+	_, store := SetUpSQLite(t, tempBaseDir)
+	defer os.RemoveAll(tempBaseDir)
+
+	ts := data.SignedTimestamp{
+		Signatures: make([]data.Signature, 0),
+		Signed: data.Timestamp{
+			Type:    data.TUFTypes["timestamp"],
+			Version: 0,
+			Expires: data.DefaultExpires("timestamp"),
+		},
+	}
+	j, err := json.Marshal(&ts)
+	require.NoError(t, err)
+	update := MetaUpdate{
+		Role:    data.CanonicalTimestampRole,
+		Version: 1,
+		Data:    j,
+	}
+	checksumBytes := sha256.Sum256(j)
+	checksum := hex.EncodeToString(checksumBytes[:])
+
+	store.UpdateCurrent("gun", update)
```

Comment:
Similarly here - could we add a second timestamp?  That way, if someone accidentally deletes the checksum line in the function, this test will fail?